src/rdkafka.h

```diff
@@ -4853,11 +4905,22 @@ struct rd_kafka_group_list {
         int group_cnt;                      /**< Group count */
 };
 
+/**
+ * @brief Options for rd_kafka_list_consumer_groups
+ */
+typedef struct rd_kafka_list_consumer_groups_options_s
+    rd_kafka_list_consumer_groups_options_t;
```

Comment:
If the type is specific to a certain Admin API then use the CamelCase format to match the API name for the type, e.g.; rd_kafka_ListConsumerGroups_options_t;

If the type is more generic, such as topic_result_t, then use snake_case.

Reply:
dito